src/backend/src/waiting-room/waiting-room.sse.service.ts

```diff
@@ -5,7 +5,7 @@ import { GameService } from 'src/game/game.service';
 import { Client } from 'src/sse/sse.service';
 
 type WaitingRoomClientMetadata = { playerId: number; emotionId: number };
-const FIFTEEN_MINUTES = 900000;
+const FIFTEEN_MINUTES = 180000;
```

Comment:
Can we rename this something like 'waiting room time'? Just so if we ever have to change it again, it makes sense